pkg/cmdutil/constants.go

```diff
@@ -3,4 +3,10 @@ package cmdutil
 const (
 	// The default indentation to use when printing data to stdout
 	DefaultJSONIndent = "    "
+
+	// DefaultPageSize is the default number of items per page when using list commands
+	DefaultPageSize = 10
+
+	// DefaultPageNumber is the default page number when using list commands
+	DefaultPageNumber = 1
```

Comment:
I think these needs to to be `var`s to be configurable at build-time.

Reply:
Build variables can only be a string 🤔  Leave it like this for now and we can figure out how to configure it later (leave a TODO)

Reply:
Checked now it doesnt work as expected! It needs to be addressed in a follow-up.